modules/ROOT/pages/changes-deprecations-removals.adoc

```diff
@@ -0,0 +1,601 @@
+:description: Page contains lists of procedures, configuration settings, and metrics removed or deprecated in Neo4j 2025. Also, you can find information on changed defaults and new functionality of neo4j-admin commands.
+
+//Check Mark
+:check-mark: icon:check[]
+
+
+[[removals-deprecations-2025]]
+= Changes, deprecations, and removals in Neo4j 2025
+
+
+== Breaking changes in Neo4j 2025.01
+
+The section covers breaking changes since the Neo4j 5.26 LTS release.
+For guidelines on how to migrate from previous Neo4j versions to 2025.01, see the link:https://neo4j.com/docs/upgrade-migration-guide/current/version-2025[Upgrade and Migration Guide].
+
+[role=label--enterprise]
+=== Clustering
+
+Discovery service::
+In 2025.01, discovery service v1 is removed.
+Therefore, transitioning from v1 to v2 must be completed before upgrading to Neo4j 2025.01.
++
+For more details, refer to the link:https://neo4j.com/docs/operations-manual/5/clustering/setup/discovery/#clustering-discovery-v1-to-v2[Operations Manual -> Moving from discovery service v1 to v2].
+
+Configuration settings::
+. In 2025.01, the settings `server.discovery.advertised_address` and `server.discovery.listen_address` are removed.
++
+To list the discovery endpoints (e.g., for providing them to the discovery resolver), use the value from the `server.cluster.advertised_address` setting.
++
+. Two configuration settings related to the discovery service v2 are renamed.
+You can still use them when migrating from Neo4j 5.26 to the 2025.01 version.
+However, it is recommended to update your _neo4j.conf_ file to use the new settings.
++
+[options=header, cols="3,3"]
+|===
+| Old name
+| New name
+
+|`dbms.cluster.discovery.v2.endpoints` 
+|`dbms.cluster.endpoints`
+
+|`dbms.kubernetes.discovery.v2.service_port_name`
+|`dbms.kubernetes.discovery.service_port_name`
+|===
+
+Ports::
+Port `5000` is no longer used from 2025.01 onwards.
+Use port `6000` for the internal traffic.
++
+For more details, refer to the xref:configuration/ports.adoc[Configuration -> Ports].
+
+=== Built-in procedures
+
+.See procedures removed in Neo4j 2025.01 without replacement.
```

Comment:
```suggestion
.See removed procedures without replacement.
```

Reply:
Honestly, I don't see much harm in mentioning the exact Neo4j version in the table headings

Reply:
I am more for consistency in this case. So, either have it everywhere or nowhere. On the other hand, having it everywhere might look a bit overloaded, hence my suggestion to remove it from all table titles.